docs/md/dev-wgs.md

```diff
@@ -50,3 +52,67 @@ The `unmapped_bam_suffix`
 is almost always `.unmapped.bam`,
 so that is its default value
 unless it is specified.
+
+The `reference_fasta_prefix` can be used to override
+the [default value](https://github.com/broadinstitute/wfl/blob/master/api/src/wfl/references.clj#L7) used by this module: 
+"gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38"
+
+## Usage
+
+###Create Workload: `/api/v1/create`
+Creates a WFL workload. Before processing, confirm that the WFL and Cromwell service accounts have
+at least read access to the input files.
+```
+curl --location --request POST 'https://dev-wfl.gotc-dev.broadinstitute.org/api/v1/create' \
+--header 'Authorization: Bearer '$(gcloud auth print-access-token) \
+--header 'Content-Type: application/json' \
+--data-raw '{
+  "cromwell": "https://cromwell-gotc-auth.gotc-dev.broadinstitute.org",
+  "input": "gs://broad-gotc-dev-wfl-ptc-test-inputs/single_sample/plumbing/truth",
+  "output": "gs://broad-gotc-dev-wfl-ptc-test-outputs/wgs-test-output/",
+  "pipeline": "ExternalWholeGenomeReprocessing",
+  "project": "PO-1234",
+  "items": [{
+          "input_cram": "develop/20k/NA12878_PLUMBING.cram",
+          "sample_name": "TestSample1234"
+          }]}'
+```
```

Comment:
What does hitting `api/v1/create` return?

Reply:
I added responses to the docs!

Reply:
thanks so much!